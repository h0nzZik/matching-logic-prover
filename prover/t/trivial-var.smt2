(declare-const x Int)
(assert (not ( = x x)))
(check-sat)