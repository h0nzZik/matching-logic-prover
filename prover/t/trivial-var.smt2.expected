<prover>
  <k>
    \exists { Vx { Int } , .Patterns } \and ( \not ( \equals ( Vx { Int } , Vx { Int } ) ) , .Patterns )
  </k>
  <goals>
    <goal>
      active: true, id: root, parent: .
      <claim>
        \not ( \exists { Vx { Int } , .Patterns } \and ( \not ( \equals ( Vx { Int } , Vx { Int } ) ) , .Patterns ) )
      </claim>
      <strategy>
        success
      </strategy>
      <trace>
        smt-cvc4
      </trace>
    </goal>
  </goals>
  <declarations>
    .DeclarationCellSet
  </declarations>
</prover>
